crates/tools/src/main.rs

```diff
@@ -11,6 +11,10 @@ mod update;
 
 use update::Mode;
 
+// Matches `edr_napi`. Important for scenarios.
+#[global_allocator]
+static ALLOC: mimalloc::MiMalloc = mimalloc::MiMalloc;
```

Comment:
Added this to match allocator used by `edr_napi` when executing scenarios from Rust